.github/workflows/developer_onboarding_notification.yml

```diff
@@ -0,0 +1,73 @@
+
+name: PR Merge Comment
```

Comment:
I'm thinking of a more descriptive name, what do you think @Mayank77maruti? 
```suggestion
name: Celebrating Initial Contributions
```

Reply:
Yup a more descriptive name would be better.